Assets/Scripts/PlayerSupervisor.cs

```diff
@@ -61,10 +61,21 @@ public class PlayerSupervisor : MonoBehaviour
     [Range(0f, 5f)]
     public float ceilingReboundAngle = 0f;
 
+// ---------------------------------------------------------
+    
+    [SerializeField] GameObject blockGameObject;
+    [SerializeField] GameObject trainingBlocksGroup;
+
+    [SerializeField] MultiBlockCreator multiBlockCreator;
```

Comment:
Try to keep all the object references at the top of the class to keep it tidy.

Reply:
Makes sense, cleaned this up in PS.